### DISCUSSION TITLE
Production app down due to upgrade

### DISCUSSION THREAD

**Discussion by elevano2024** [hasura/graphql-engine#10514] [General]
Hi i just upgraded my hasura instance and now i am seeing this issue below i need to access my account to reload DB right now it is out of reach help me fix this ASAP

![image](https://github.com/user-attachments/assets/667b16ab-382a-440e-9071-b8e7dde0065a)

Inconsistent object: source "default" is inconsistent because of stale database introspection is used. The source couldn't be reached for a fresh introspection because we got error: connection error
connection to server at "ep-fancy-unit-a6sygnaj.us-west-2.aws.neon.tech" (35.84.214.33), port 5432 failed: ERROR: Your project has exceeded the data transfer quota. Upgrade your plan to increase limits.

**Comment by meetzaveri** [comment]
Hi @elevano2024 

This is probably due to Hasura unable to reach to your DB. From the error, it seems there were limitations from [NeonDB](https://neon.tech/) project you were using. Please see this relevant Neon DB doc which reflects the issue and remedies offered by them - https://neon.tech/docs/introduction/usage-metrics#data-transfer

I also saw that you had opened a ZD ticket with us, and we suggested you to reach out to NeonDB support in that. Is this issue resolved ? If yes, can we close the discussion ? If not, let us know in case. Thanks!
